controllers/ChallengeController.php

```diff
@@ -174,4 +175,24 @@ public function actionEdit()
             ]
         );
     }
+    public function actionReviewFunding($id, $status)
+    {
+        $model = Funding::findOne(['id' => $id]);
+
```

Comment:
missign a 404 NOT_FOUND handling , what if the giving  `id` doesn't match any funding ?